builder/cloudstack/step_create_instance.go

```diff
@@ -161,11 +162,13 @@ func (s *stepCreateInstance) Cleanup(state multistep.StateBag) {
 			return
 		}
 
-		ui.Error(fmt.Sprintf("Error destroying instance: %s", err))
+		ui.Error(fmt.Sprintf("Error destroying instance. Please destroy it manually.\n\n"+
+			"\tName: %s\n"+
+			"\tError: %s", config.InstanceName, err))
+		return
```

Comment:
IMHO I still think we should retry without expunge. You can see it as two actions _mark for deletion_ and _remove now_. If we always can do the first we should do it. It ensures that no instances are laying arround.